radicle-cli/examples/rad-remote.md

```diff
@@ -0,0 +1,23 @@
+Now lets add a bob as a new remote
+
+```
+$ rad remote add did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --name bob
+✓ Remote bob added with rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
+```
+
+Now, we can see that there is a new new remote in the list of remotes
+
+```
+$ rad remote list
+❲fetch❳ bob z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
+❲fetch❳ rad z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi
```

Comment:
Yeah this is good, though the 4th column is a redundant in my opinion, and can be obtained via `git remote -v`. I'd remove it, or if you want to keep it, I'd hide it behind a `-v` flag.

Reply:
Yeah I see your point and I agree! I will remove the url